scalapb/scalapb_2.12/build.gradle

```diff
@@ -49,7 +49,8 @@ tasks.withType(Test) {
         // when testing scalapb module with Java 8. This bug was fixed in Java 9.
         // - https://github.com/gradle/gradle/issues/8432
         // - https://bugs.openjdk.java.net/browse/JDK-8057919
-        if (System.env.TEST_JAVA_VERSION == '8') {
+        if (rootProject.hasProperty('testJavaVersion') &&
+                rootProject.findProperty('testJavaVersion') == '8') {
```

Comment:
`hasProperty` is redundant? `findProperty()` will return `null` when the property is missing.

Reply:
Fixed! It's much simplified now. 😄